scripts/xcodebuild.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -eo pipefail
+
+workspace=$1
+action=$2
+platform=$(grep -o '\w\+/ReactTestApp.xcodeproj' "$workspace/contents.xcworkspacedata")
+
+if [[ $platform == ios/* ]]; then
+  device_name=${3:-'iPhone 11'}
+  device=$(instruments -s devices 2> /dev/null | grep "${device_name} (")
```

Comment:
FYI On my machine `xcrun simctl list devices 'iPhone 11' available [--json]` is on average 1.1 sec faster.

Reply:
TIL. I'm trying to get the OS version but `simctl` returns `13.4` when we need `13.4.1`. Is there a way to make it return the latter?
```
Command line invocation:
    /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -workspace ios/Example.xcworkspace -scheme ReactTestApp -destination "platform=iOS Simulator,name=iPhone 11,OS=13.4" "-skip-testing:ReactTestAppTests/ReactNativePerformanceTests" CODE_SIGNING_ALLOWED=NO COMPILER_INDEX_STORE_ENABLE=NO

User defaults from command line:
    IDETestRunSkipIdentifiers = (
    "ReactTestAppTests/ReactNativePerformanceTests"
)

Build settings from command line:
    CODE_SIGNING_ALLOWED = NO
    COMPILER_INDEX_STORE_ENABLE = NO

xcodebuild: error: Unable to find a destination matching the provided destination specifier:
		{ platform:iOS Simulator, OS:13.4, name:iPhone 11 }

	The requested device could not be found because no available devices matched the request.

	Available destinations for the "ReactTestApp" scheme:
		{ platform:iOS Simulator, id:D811C5AA-C2AF-4B03-90EF-AB545B406658, OS:13.4.1, name:iPad (7th generation) }
		{ platform:iOS Simulator, id:568EB94B-03F3-4AB4-9F67-1CA05E4C8387, OS:13.4.1, name:iPad Air (3rd generation) }
		{ platform:iOS Simulator, id:C1215F09-3FCE-4A6E-B9E1-F5188AA02119, OS:13.4.1, name:iPad Pro (9.7-inch) }
		{ platform:iOS Simulator, id:DB6CF52F-DE36-43F9-8E38-509154AA0969, OS:13.4.1, name:iPad Pro (11-inch) (2nd generation) }
		{ platform:iOS Simulator, id:C8544BF4-3BE7-41D0-BD9C-D3E0C301B3A8, OS:13.4.1, name:iPad Pro (12.9-inch) (4th generation) }
		{ platform:iOS Simulator, id:7E6E4967-D216-4BD6-A7A1-C028D211F758, OS:13.4.1, name:iPhone 8 }
		{ platform:iOS Simulator, id:F2E0AB84-EC77-4A67-8B57-0AAFD176A61C, OS:13.4.1, name:iPhone 8 Plus }
		{ platform:iOS Simulator, id:4312DEBE-4CCE-49B6-8E7E-6DAEC1E3AF34, OS:13.4.1, name:iPhone 11 }
		{ platform:iOS Simulator, id:39B1B54A-0540-4AD3-A3A4-3C0129B95A2E, OS:13.4.1, name:iPhone 11 Pro }
		{ platform:iOS Simulator, id:83992BFD-C517-4F2B-BB8E-0555AF68A31C, OS:13.4.1, name:iPhone 11 Pro Max }
		{ platform:iOS Simulator, id:A4CEE87B-AC3B-4F70-A0E6-D1E673AF72B6, OS:13.4.1, name:iPhone SE (2nd generation) }

	Ineligible destinations for the "ReactTestApp" scheme:
		{ platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Generic iOS Device }
		{ platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder, name:Generic iOS Simulator Device }
```

Reply:
How about using the UUID as destination? iirc those include the OS

```
-destination 'platform=iOS Simulator,id=YOUR_PHONE_UUID'
```